lib/server.js

```diff
@@ -189,3 +203,21 @@ module.exports = {
   }
 };
 
+function cleanUpAndExit() {
+  if (electronPids) {
```

Comment:
If you look into the change above, you'd realise that I'm keeping all the spawned electron PIDs in a global variables. The reason for this `cleanUpAndExit()` is mainly to force clean up all the running electron PIDs that didn't get terminated/killed when we send `SIGTERM`/`Ctrl + C` to this process itself.